msm/__main__.py

```diff
@@ -73,7 +73,12 @@ def add_search_args(subparser):
     add_search_args(subparsers.add_parser('info'))
     subparsers.add_parser('list').add_argument('-i', '--installed',
                                                action='store_true')
-    subparsers.add_parser('update')
+
+    def add_optional_search_args(subparser):
```

Comment:
How about instead make this part of `add_search_args` (maybe a `skill_is_optional=False` parameter or something)